focus-android/tools/docker/Dockerfile

```diff
@@ -20,8 +20,7 @@ RUN apt-get update -qq \
     # We need to install tzdata before all of the other packages. Otherwise it will show an interactive dialog
     # which we cannot navigate while building the Docker image.
     && apt-get install -y tzdata \
-    && apt-get install -y openjdk-8-jdk \
-                          openjdk-11-jdk \
+    && apt-get install -y openjdk-11-jdk \
```

Comment:
Hm we're still using a JAVA8PATH above: https://github.com/mozilla-mobile/firefox-android/pull/1077/files#diff-d33eee172ac4faa57c5063f0cbfbf224d0ff79d7d7dc6e917bbd3e12ebfd0df5L17

Need to look into this.

Reply:
OK, updated this. 🤞